pkg/kubelet/config/flags.go

```diff
@@ -51,11 +49,9 @@ type ContainerRuntimeOptions struct {
 // AddFlags adds flags to the container runtime, according to ContainerRuntimeOptions.
 func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	// General settings.
-	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible value: 'remote'.")
```

Comment:
there are still a few mentions of `--container-runtime` in codebase, for example in utils.sh and local-up.sh. Can you please confirm they are still needed and add comment there why or clean up. I remember last time we tried to remove those some of the tests started failing

Reply:
Find another 3 places and I updated the PR.
1. cmd/kubelet/app/options/options.go: `This flag can only be used with --container-runtime=remote.`
2. cluster/gce/util.sh
3. hack/local-up-cluster.sh